playground/samples/customArray.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+
+const ArrayFieldTemplate = props => (
```

Comment:
Please use a regular `function` declaration here, for consistency with the rest of the code in this project.